docs/administrator_guide/reporting/notifications.md

```diff
@@ -64,17 +72,16 @@ System Health Report: `SystemHealthReport`
 
 - ExcludedTraceTypes - *Non-Alerted Document Review Report Only* - an optional field that allows for the exclusion of communications of a certain `Trace Type` from a report. Multiple values can be excluded if comma delimited. `["Office 365 Email", "Slack]`
 - IncludeDetails - *System Health Report Only* - a true/false value which pro. Currently only used by the SystemHealthAlertsReport to provide extra details on the error each Task encountered
+- UtcEndOfDay - *Reconciliation Report Only* - the UTC time your team treats as End-of-Day. This is used to separate data in the report into days on which the data was retrieved.
+- Days - *Reconciliation Report Only* - an optional number of days to include in each report, by default 7. You will still receive the report at the frequency set above.
 
 *Example JSON Configuration*
 
 ```json
 {"NonAlertedDocumentReviewReport":{"Enabled":true,"Recipients":"user@domain.com","FrequencyInMinutes":1440,"EmailFrom":"noreply@relativity.one", "ExcludedTraceTypes":"Slack"},
-"RuleChangeReport": {"Enabled":true,"Recipients":"user@domain.com","FrequencyInMinutes":1440,"EmailFrom":"noreply@relativity.one"}, "SystemHealthReport": {"Enabled":true,"Recipients":"user@domain.com","FrequencyInMinutes":1440,"EmailFrom":"noreply@relativity.one"}}
+"RuleChangeReport": {"Enabled":true,"Recipients":"user@domain.com","FrequencyInMinutes":1440,"EmailFrom":"noreply@relativity.one"}, "SystemHealthReport": {"Enabled":true,"Recipients":"user@domain.com","FrequencyInMinutes":1440,"EmailFrom":"noreply@relativity.one"}, "ReconciliationReport": {"Enabled": true,"Recipients":"user@domain.com","FrequencyInMinutes":1440,"EmailFrom":"noreply@relativity.one","UtcEndOfDay":"17:00:00","Days":7}}
 ```
 
-#### Example JSON Configuration with All Reports
-![](media/notifications/Updated Configuration.PNG)
-
 - If there is nothing to report an email will not be sent, but an Automated Report object will still be created within the product
 - The Non-Alerted Document Review Report only analyzes events that occurred the previous day, and cannot gather historical results
 - To see what the report looks like in Trace, see the [User Guide]({{ site.baseurl }}{% link docs/user_guide/reporting/notifications.md %})
```

Comment:
Can we please add a screenshot of an actual reconciliation report to the referenced  page: https://relativitydev.github.io/relativity-trace-documentation/docs/user_guide/reporting/notifications.html